sklearn/metrics/__init__.py

```diff
@@ -1,172 +1,173 @@
-from ._regression import mean_squared_log_error
-from ._regression import median_absolute_error
-from ._regression import mean_absolute_percentage_error
-from ._regression import r2_score
-from ._regression import mean_tweedie_deviance
-from ._regression import mean_poisson_deviance
-from ._regression import mean_gamma_deviance
-
-
-from ._scorer import check_scoring
-from ._scorer import make_scorer
-from ._scorer import SCORERS
-from ._scorer import get_scorer
-
-from ._plot.det_curve import plot_det_curve
-from ._plot.det_curve import DetCurveDisplay
-from ._plot.roc_curve import plot_roc_curve
-from ._plot.roc_curve import RocCurveDisplay
-from ._plot.precision_recall_curve import plot_precision_recall_curve
-from ._plot.precision_recall_curve import PrecisionRecallDisplay
-
-from ._plot.confusion_matrix import plot_confusion_matrix
-from ._plot.confusion_matrix import ConfusionMatrixDisplay
-
-
-__all__ = [
-    'accuracy_score',
-    'adjusted_mutual_info_score',
-    'adjusted_rand_score',
-    'auc',
-    'average_precision_score',
-    'balanced_accuracy_score',
-    'calinski_harabasz_score',
-    'check_scoring',
-    'classification_report',
-    'cluster',
-    'cohen_kappa_score',
-    'completeness_score',
-    'ConfusionMatrixDisplay',
-    'confusion_matrix',
-    'consensus_score',
-    'coverage_error',
-    'dcg_score',
-    'davies_bouldin_score',
-    'DetCurveDisplay',
-    'det_curve',
-    'euclidean_distances',
-    'explained_variance_score',
-    'f1_score',
-    'fbeta_score',
-    'fowlkes_mallows_score',
-    'get_scorer',
-    'hamming_loss',
-    'hinge_loss',
-    'homogeneity_completeness_v_measure',
-    'homogeneity_score',
-    'jaccard_score',
-    'label_ranking_average_precision_score',
-    'label_ranking_loss',
-    'log_loss',
-    'make_scorer',
-    'nan_euclidean_distances',
-    'matthews_corrcoef',
-    'max_error',
-    'mean_absolute_error',
-    'mean_squared_error',
-    'mean_squared_log_error',
-    'mean_poisson_deviance',
-    'mean_gamma_deviance',
-    'mean_tweedie_deviance',
-    'median_absolute_error',
-    'mean_absolute_percentage_error',
-    'multilabel_confusion_matrix',
-    'mutual_info_score',
-    'ndcg_score',
-    'normalized_mutual_info_score',
-    'pair_confusion_matrix',
-    'pairwise_distances',
-    'pairwise_distances_argmin',
-    'pairwise_distances_argmin_min',
-    'pairwise_distances_chunked',
-    'pairwise_kernels',
-    'plot_confusion_matrix',
-    'plot_det_curve',
-    'plot_precision_recall_curve',
-    'plot_roc_curve',
-    'PrecisionRecallDisplay',
-    'precision_recall_curve',
-    'precision_recall_fscore_support',
-    'precision_score',
-    'r2_score',
-    'rand_score',
-    'recall_score',
-    'RocCurveDisplay',
-    'roc_auc_score',
-    'roc_curve',
-    'SCORERS',
-    'silhouette_samples',
-    'silhouette_score',
-    'top_k_accuracy_score',
-    'v_measure_score',
-    'zero_one_loss',
-    'brier_score_loss',
-]
+"""

```

Comment:
This is a lot of diff for no diff :)

Reply:
Yes, windows and \r, a very long story of very annoying mistakes.

Reply:
Fixing them...

Reply:
It turns out the sources are using \r... I need to add them back.

Reply:
It is not the same eol everywhere... Anywway, I removed the wrong differences.

Reply:
It seems good now.